pkgs/development/libraries/libevent/default.nix

```diff
@@ -13,6 +13,17 @@ stdenv.mkDerivation rec {
     sha256 = "1hhxnxlr0fsdv7bdmzsnhdz16fxf3jg2r6vyljcl3kj6pflcap4n";
   };
 
+  patches = [
+    (fetchurl {
+      url = "https://github.com/libevent/libevent/commit/22dd14945c25600de3cf8b91000c66703b551e4f.patch";
```

Comment:
These patches are generated dynamically by GitHub, so `fetchpatch` needs to be used.